From ad1bc1912fa820321e4004338143aa79221274f4 Mon Sep 17 00:00:00 2001
From: Fei Jiang <fei.jiang@intel.com>
Date: Fri, 29 Dec 2017 19:14:16 +0800
Subject: [PATCH 0486/1240] drm/i915/gvt: hard code Pipe B plane owner to UOS

It is a work around patch due to plane restriction patches are not porting

Change-Id: If09ff8c40254ec275dc2d9b9674d7267d306a7e7
Signed-off-by: Fei Jiang <fei.jiang@intel.com>
Reviewed-by: Dong, Eddie <eddie.dong@intel.com>
Tested-by: Dong, Eddie <eddie.dong@intel.com>
---
 drivers/gpu/drm/i915/gvt/cwpgt.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/drivers/gpu/drm/i915/gvt/cwpgt.c b/drivers/gpu/drm/i915/gvt/cwpgt.c
index aae1994..4e8ef1a 100644
--- a/drivers/gpu/drm/i915/gvt/cwpgt.c
+++ b/drivers/gpu/drm/i915/gvt/cwpgt.c
@@ -405,6 +405,13 @@ static int cwpgt_sysfs_add_instance(struct cwpgt_hvm_params *vp)
 	struct cwpgt_hvm_dev *info;
 
 	struct intel_vgpu_type type = cwpgt_priv.gvt->types[0];
+
+	/* todo: wa patch due to plane restriction patches are not porting */
+	cwpgt_priv.gvt->pipe_info[1].plane_owner[0] = 1;
+	cwpgt_priv.gvt->pipe_info[1].plane_owner[1] = 1;
+	cwpgt_priv.gvt->pipe_info[1].plane_owner[2] = 1;
+	cwpgt_priv.gvt->pipe_info[1].plane_owner[3] = 1;
+
 	type.low_gm_size = vp->aperture_sz * VMEM_1MB;
 	type.high_gm_size = (vp->gm_sz - vp->aperture_sz) * VMEM_1MB;
 	type.fence = vp->fence_sz;
-- 
2.7.4

